chart/jenkins-operator/templates/jenkins.yaml

```diff
@@ -48,6 +48,18 @@ spec:
     recoveryOnce: {{ .Values.jenkins.backup.recoveryOnce }}
     {{- end }}
   {{- end }}
+  {{- with .Values.jenkins.services.http }}
```

Comment:
@tomaszsek 
@jakalkhalili 

Would it not be better to just do the following? 
```
  {{- with .Values.jenkins.service }}
  service: {{- toYaml . | nindent 4 }}
  {{- end }}
  {{- with .Values.jenkins.slaveService }}
  slaveService: {{- toYaml . | nindent 4 }}
  {{- end }}
```

Reply:
@snooyen Looks much better thanks.